PatreonClient/PatreonClient.csproj

```diff
@@ -4,8 +4,17 @@
         <TargetFramework>netstandard2.1</TargetFramework>
     </PropertyGroup>
 
+	<ItemGroup>
```

Comment:
A little bit uneasy about exposing internals to tests, I would prefer not to have this.